docs/source/certificates/certificates.md

```diff
@@ -0,0 +1,173 @@
+## Certificate Management Guide
+
+**Audience**: Fabric network admins
+
+This guide provides a brief overview, resources, and technical details for Fabric network administrators to manage certificates (certs).
+
+### Key Concepts
+
+**Registration** – A registration is a username (enrollment ID) and password stored in the certificate authority (CA). This is created by a CA admin level user and has no expiration. The registration also contains any roles and attributes required.
+
+**Enrollment** – Based on the registration username and password, an enrollment (or identity), can be created. The enrollment contains a public/private key pair and the (encoded) roles and attributes from the registration. The public key in the enrollment is referred to as a certificate, which is a signed certificate that is required to use Hyperledger Fabric.
+
+### Certificate types
+
+Hyperledger Fabric implements two basic types of certificates: 1) **enrollment** (or identity) certificates, and 2) **TLS** certificates.
+
+#### Enrollment Certificates
+
+Enrollment certificates are classed by type into the following four roles:
+
+* **admin**
+* **peer**
+* **orderer**
+* **client**
+
+The roles are used for the following purposes:
+
+<b>admin:</b> Signed certificates used by Hyperledger Fabric to identify identities that are administrative-level user certificates. These identities are required in order to make changes to Hyperledger Fabric configurations.
+
+<b>peer and orderer:</b> Signed certificates used by Hyperledger Fabric nodes (peers and orderers) and are node enrollment certificates. These certificates are either located physically on the node or mapped so they are visible to the node. Hyperledger Fabric nodes must have a valid certificate with the required attributes in order for the node to start.
+
+<b>client:</b> Signed certificates that allow signed requests passed to the Hyperledger Fabric nodes. These can be requests that do not require admin level certificates or client applications that need to pass signed requests to Hyperledger Fabric.
+
+#### TLS Certificates
+
+TLS certificates are signed certificates that allow Hyperledger Fabric nodes and clients to sign and encrypt communications. A valid TLS certificate is required for all communications.
+
+### Certificate Expiration
```

Comment:
It feels like we should add a section about revocation - wdyt?

Reply:
Three *Rs* in ceRtificate lifecycle: 

- Renewal 
- Recovery 
- Revocation

Reply:
TBA. Initial iterations had no recovery and no revocation.